profile_library/bosch/Smart Home Controller 1 Gen/model.json

```
@@ -0,0 +1,22 @@
+{
+  "description": "The measured Bosch Smart Home Controller had the following devices connected: four Bosch Smart Home Radiator Thermostat II (TRV_GEN2), two Bosch Smart Home Radiator Thermostat I (TRV), and two Bosch Smart Home Door/Window Contact II (SWD2).During the 120-second measurement period, the temperature was adjusted both directly on the device itself and via the Smart Home Controller on one TRV thermostat and on one TRV_GEN2 thermostat. In addition, both window sensors were triggered twice each by opening and closing the windows.",
+  "measure_method": "script",
+  "measure_device": "Tapo P115",
+  "measure_device_firmware": "1.3.4 Build 250403 Rel.150504",
+  "name": "Bosch Smart Home Controller 1 Gen.",
+  "aliases": [
+    "SHC1"
```

```suggestion
    "SHC1",
    "SmartHomeController"
```

Alias has to be added, as this is how it's identified in HA looking at the information of the device page you shared.